src/components/History.tsx

```diff
@@ -15,6 +15,15 @@ const History = (props: Props): JSX.Element => {
               .toFormat('d LLLL y')}
           </td>
         </tr>
+        <tr className="govuk-table__row">
+          <td className="govuk-table__cell"></td>
+          <td className="govuk-table__cell">Retention expires</td>
+          <td className="govuk-table__cell">
+            {props.documentSubmission.retentionExpiresAt
+              .toLocal()
+              .toFormat('d LLLL y')}
```

Comment:
Because we're using the same formatter here and for _Valid until_, we should probably introduce a function to wrap `.toLocal().toFormat('d LLLL y')`

Reply:
👍